packages/gql/src/balancer/pools.ts

```diff
@@ -195,12 +195,14 @@ export const aprPools = gql`
     $minTvl: BigDecimal
     $maxTvl: BigDecimal
     $block: Int
+    $tokensAsVunerabilityAffectedPools: [String!]
   ) {
     pools(
       where: {
         createTime_lte: $createdBefore
         totalLiquidity_gte: $minTvl
         totalLiquidity_lte: $maxTvl
+        tokens_: { address_not_in: $tokensAsVunerabilityAffectedPools }
```

Comment:
So we're not going to report this even historically?

Reply:
🫠 you are right

Reply:
added a tech debt here: https://linear.app/bleu-llc/issue/BAL-1165/handle-bb-pools-proportion-for-yield-apr 
talked to Fabio and the majority of bb pools have token yield, and the token apr will never be calculated without the price/proportion

So will filter out those pools and add them back into this task